.github/workflows/turbo_hierarchy.yml

```diff
@@ -0,0 +1,59 @@
+name: Turborepo Hierarchy
+
+on:
+  push:
+    branches:
+      - "develop"
+      - "devnet"
+      - "testnet"
+      - "mainnet"
+      - "releases/iota-*-release"
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+
+jobs:
+  diff:
+    runs-on: [ubuntu-latest]
+    concurrency:
+      group: turbo-diff-${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+      cancel-in-progress: ${{ github.ref != 'refs/heads/develop' }}
+    outputs:
+      isRpc: ${{ steps.diff.outputs.isRpc }}
+      isWallet: ${{ (steps.turbo.outputs.packages && contains(fromJson(steps.turbo.outputs.packages), 'iota-wallet')) }}
+      isExplorer: ${{ (steps.turbo.outputs.packages && contains(fromJson(steps.turbo.outputs.packages), 'iota-explorer')) }}
+      isTypescriptSDK: ${{ (steps.turbo.outputs.packages && contains(fromJson(steps.turbo.outputs.packages), '@iota/iota-sdk')) }}
+      isAppsBackend: ${{ (steps.turbo.outputs.packages && contains(fromJson(steps.turbo.outputs.packages), 'apps-backend')) }}
+      isAppsUiKit: ${{ (steps.turbo.outputs.packages && contains(fromJson(steps.turbo.outputs.packages), '@iota/apps-ui-kit')) }}
+      isGraphQlTransport: ${{ (steps.turbo.outputs.packages && contains(fromJson(steps.turbo.outputs.packages), '@iota/graphql-transport')) }}
+      isLedgerjs: ${{ (steps.turbo.outputs.packages && contains(fromJson(steps.turbo.outputs.packages), '@iota/ledgerjs-hw-app-iota')) }}
+    steps:
+      - uses: actions/checkout@eef61447b9ff4aafe5dcd4e0bbf5d482be7e7871 # v4.2.1
+      - name: Detect Changes (turbo)
+        uses: "./.github/actions/turbo-diffs"
+        id: turbo
+      - name: Detect Changes (diff)
+        uses: "./.github/actions/diffs"
+        id: diff
+
+  turborepo:
+    uses: ./.github/workflows/_turborepo.yml
```

Comment:
Could this not be turned into the hierarchy instead?

Reply:
What do you mean?

Reply:
Can we not merge the turborepo and turporepo_hierarchy files?

Reply:
I suppose we could, yes

Reply:
Can I do that in a separate PR?

Reply:
I think it makes more sense here but as you want

Reply:
@DaughterOfMars if you are not doing it here, please open an issue

Reply:
https://github.com/iotaledger/iota/issues/5072